src/input/process/flow.rs

```diff
@@ -142,27 +129,18 @@ where
     // Validate flows
     for (process_id, map) in map.iter() {
         let process = processes.get(process_id).unwrap();
-        validate_process_flows_map(process, map, milestone_years)?;
+        validate_process_flows_map(process, map)?;
     }
 
     Ok(map)
 }
 
 /// Validate flows for a process
-fn validate_process_flows_map(
-    process: &Process,
-    map: &ProcessFlowsMap,
-    milestone_years: &[u32],
-) -> Result<()> {
+fn validate_process_flows_map(process: &Process, map: &ProcessFlowsMap) -> Result<()> {
     let process_id = process.id.clone();
-    let year_range = &process.years;
-    let reference_years: HashSet<u32> = milestone_years
-        .iter()
-        .copied()
-        .filter(|year| year_range.contains(year))
-        .collect();
+    let reference_years = &process.years;
     let reference_regions = &process.regions;
-    for year in &reference_years {
+    for year in &reference_years.clone() {
```

Comment:
Think this could just be:

```suggestion
    for year in reference_years.iter() {
```